sklearn/README.md

```diff
@@ -0,0 +1,30 @@
+# Cloud ML Engine samples for scikit-learn
+
+This repository contains samples that demonstrate how to use Cloud ML Engine for
+training and prediction with scikit-learn.
+
+The samples are available as notebooks, Python files, and
+[documentation](https://cloud.google.com/ml-engine/docs/scikit/tutorials).
+- - -
+
+## About the data
+
+These samples use the [Census income data set](https://archive.ics.uci.edu/ml/datasets/census+income)
+from the UCI Machine Learning Repository.
+
+This dataset is provided by a third party. Google provides no representation,
+warranty, or other guarantees about the validity or any other aspects of this
+dataset.
+
+## Samples
+
+[Training](census-training) - Shows how to use Cloud ML Engine to train a
+scikit-learn model.
+
+[Prediction](census-prediction) - Shows how to use Cloud ML Engine to serve
```

Comment:
(census-training) doesn't work as a link. (I'm not sure how to get the shorthand to work)

Direct links work.